src/AutorestSwift/Code Generator/SwiftGenerator.swift

```diff
@@ -92,4 +105,20 @@ class SwiftGenerator: CodeGenerator {
             try object.toFile(inFolder: modelUrl)
         }
     }
+
+    private func generateChoice(for choice: ChoiceSchema) throws {
+        try renderTemplate(filename: "ChoiceSchema.stencil", dictionary: ["choice": choice])
+    }
+
+    private func generateObject(for object: ObjectSchema) throws {
+        try renderTemplate(filename: "ObjectSchema.stencil", dictionary: ["object": object])
+    }
+
+    private func renderTemplate(filename: String, dictionary: [String: Any]) throws {
+        let fsLoader = FileSystemLoader(bundle: [Bundle.main])
+        let environment = Environment(loader: fsLoader)
```

Comment:
We can store the FileSystemLoader and Environment variables as private variables of the SwiftGenerator so you don't need to instantiate them repeatedly.